src/main/main.ts

```diff
@@ -11,16 +11,33 @@
 import path from 'path';
 import { app, BrowserWindow, shell } from 'electron';
 import { autoUpdater } from 'electron-updater';
-import log from 'electron-log';
+import log, { LogMessage } from 'electron-log';
 import 'reflect-metadata';
 import './ipc';
+import chalk from 'chalk';
 import AppDataSource from '../data-source';
-import { resolveHtmlPath } from './util';
+import { resolveHtmlPath, getDataPath } from './util';
 import MenuBuilder from './menu';
 
 class AppUpdater {
   constructor() {
+    log.transports.file.resolvePath = () =>
+      path.join(getDataPath(), 'logs/main.log');
     log.transports.file.level = 'info';
+    log.transports.console.format = (message: LogMessage, _data: unknown) => {
+      const str = `[${message.date.toUTCString()}] ${`[${message.level}]`.padEnd(
```

Comment:
looks like this is following the format of file transport. 
docs format for console: '%c{h}:{i}:{s}.{ms}%c › {text}'

Reply:
It can be set to whatever you want